src/tokens.ts

```diff
@@ -13,17 +13,19 @@ export const TOKEN_SYMBOLS_MAP = {
       [CHAIN_IDs.MAINNET]: "0x44108f0223A3C3028F5Fe7AEC7f9bb2E66beF82F",
       [CHAIN_IDs.OPTIMISM]: "0xFf733b2A3557a7ed6697007ab5D11B79FdD1b76B",
       [CHAIN_IDs.POLYGON]: "0xF328b73B6c685831F238c30a23Fc19140CB4D8FC",
-      [CHAIN_IDs.SEPOLIA]: "0x49fCaC04AE71dbD074304Fb12071bD771e0E927A"
+      [CHAIN_IDs.SEPOLIA]: "0x49fCaC04AE71dbD074304Fb12071bD771e0E927A",
     },
+    coingeckoId: "across-protocol",
```

Comment:
ooc, is it not possible to use the contract address for this? We don't need these IDs in the bots. Admittedly, we don't require historical prices, but I _think_ cg also supports historical prices via contract address.

https://docs.coingecko.com/reference/contract-address-market-chart